addon/models/academic-year.js

```diff
@@ -1,9 +1,5 @@
 import Model, { attr } from '@ember-data/model';
-import { computed } from '@ember/object';
 
 export default Model.extend({
   title: attr('number'),
```

Comment:
This is a string now.

Reply:
:heavy_check_mark:

Reply:
Is this all sorted out? I'd be glad to merge if so.

Reply:
@dartajax  this needs the corresponding backend change to land first: https://github.com/ilios/ilios/pull/3306